conf/default.yaml

```diff
@@ -5,7 +5,9 @@ s3:
   secretAccessKey: '{{ENV.AWS_SECRET_ACCESS_KEY}}'
   region: '{{ENV.AWS_DEFAULT_REGION}}'
   profile: '{{ENV.AWS_PROFILE}}'
-
+minimumImageSize:
+  width: '{{ENV.MIN_WIDTH}}'
+  height: '{{ENV.MIN_HEIGHT}}'
```

Comment:
These need to have a default. Otherwise the code below won't work as expected if the `ENV.MIN_WIDTH` is not set. That would result in `minimumImageSize.width = undefined` and the code below (`size.width > undefined`) would result in false.